lib/cc/cli/engines/enable.rb

```diff
@@ -27,7 +26,10 @@ def pull_docker_images
         end
 
         def enable_engine
-          parsed_yaml.enable_engine(engine_name)
+          config.engines[engine_name] ||= {}
+          config.engines[engine_name].enabled = true
```

Comment:
It looks like right now our `codeclimate-yaml` gem throws an error when this is called. It's:
`NameError: uninitialized constant CC::Yaml::Parser::Ruby`

shows up in [two] (https://github.com/codeclimate/codeclimate-yaml/blob/master/lib/cc/yaml/nodes/sequence.rb#L97) [spots] (https://github.com/codeclimate/codeclimate-yaml/blob/master/lib/cc/yaml/nodes/mapping.rb#L80)

I think it's a mistake in `cc/yaml`. Looking into fixing.